src/data/projects/beep.json

```diff
@@ -13,5 +13,14 @@
     { "name": "Michael Tintuic", "twitter": "michaeltintiuc" },
     { "name": "Dmytro Kovalenko", "twitter": "tasteofdalife" },
     { "name": "Drew Griffith", "twitter": "drewmodus" }
+  ],
+  "title": "Check if you've been hacked",
+  "screenshots": [
+    {"screen": "https://res.cloudinary.com/modus-labs/image/upload/v1538144594/beep/Beep-Screenshot-iPhoneX-01.png"},
```

Comment:
Change `screen` to 
```json
"screen": "beep/Beep-Screenshot-iPhoneX-01.png"
```
^^ no URI

Then you consume it with the LazyImage component

```jsx
import Image from '../../LazyImage';

<Image filename={project.screen} path="" alt={project.name} />
```

Reply:
Nice, thanks! but the path is not set right and images are not loading. tried with 
`"screen": "beep/Beep-Screenshot-iPhoneX-01.png"` and
`"screen": "/beep/Beep-Screenshot-iPhoneX-01.png"`
but isn't catching.